ThunderCloud/QuizQuestionContainerViewController.swift

```diff
@@ -400,6 +400,8 @@ open class QuizQuestionContainerViewController: AccessibilityRefreshingViewContr
             quiz.currentIndex = quiz.currentIndex + 1
             guard let questionViewController = quiz.questionViewController() as? QuizQuestionContainerViewController else { return }
             questionViewController.quiz = quiz
+            questionViewController.hidesBottomBarWhenPushed =
```

Comment:
I have not set this below on the quiz completion

Reply:
That seems correct, the completion won't need to hide the bottom bar because it's already hidden!

Reply:
Yeah, you'd safely assume a question came first 😋